src/core/server/http/http_service.ts

```diff
@@ -92,6 +94,9 @@ export class HttpService implements CoreService<InternalHttpServiceSetup, HttpSe
     }
 
     const { registerRouter, ...serverContract } = await this.httpServer.setup(config);
+
+    registerCoreHandlers(serverContract, config, this.env);
+
```

Comment:
I did not add a test that ensure this is properly invoked during the service `setup` phase as the feature is heavily integration-tested. Tell we if we want to still add a test.